python/Ganga/new_tests/GPI/Bugs/Savannah10064.py

```diff
@@ -5,7 +5,7 @@
 
 class Savannah10064(GangaUnitTest):
 
-    def Savannah10064(self):
+    def test_Savannah10064(self):
```

Comment:
I wanted to rename those functions to be able to keep doing:
```nosetest Ganga/.../.../myTestScript.py```

If we want to drop this behaviour then I'm happy to revert this but it made life easier by being able to re-run a lightweight test without having to perform a complex setup or make a call to nosetests with too many arguments. I'm actually quite keen to fight to keep this style of workflow.